### ISSUE TITLE
[BE] 10-10EZR: Receive/Pre-fill Section III (3) (Dependent/Spouse Info) EZR Data 

### ISSUE THREAD

**Issue by alexseelig** [department-of-veterans-affairs/va.gov-team#68216]
As a Veteran, I would like to be able to update as much of my information online as I can so that I can make changes easily without having to call the hotline and wait on hold or print out the form and snail mail it in.

- [ ] Build an API that can receive all EZR fields from the ES for **Section III (3)** of the EZR (fields listed below)
- [ ] Make sure that the information we receive can be used to pre-fill the EZR Form--we will have a separate story for the FE piece
- [ ] Test API to make sure all info is received from ES
- [ ] **Success: We have an API that can receive and pre-fill EZR Section III (Dependent/Spouse Info) info from ES into our app** 

-------------------------------
EZR Fields ([spreadsheet with all EZR fields](https://docs.google.com/spreadsheets/d/1IGLA8GT35ffS2PnFvCwdnA7bWTL5187TN9QlVwUXYr4/edit#gid=1410739912)):

**Section III - Dependent Information**
Spouse's Name
Spouse's SSN
Spouse's DOB 
Date of Marriage
Do you and your spouse cohabitatate? Y/N
(if no) Spouse's Address
Dependent's Name
Dependent's DOB
Dependent's SSN
Date dependent became dependent 
Dependent's relationship to you 
Is your dependent living with a permanent disability that happened before they turned 18 years old? 
If your dependent is between 18 and 23, were they enrolled as a student in 20xx? 
Total amount of money your dependent paid for college, vocational rehabilitation or training? 
Did your dependent live with you in 20xx? (previous year)
Did your dependent earn income in 20xx? (previous year)

----------------------------
**Resources:**

- **[API Docs](https://ves.va.gov/esr-ws/spring-ws/getEESummary/eeSummary.wsdl)**
- [Lihan's Documentation](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/tutorials/record_vcr.md)
- [VA Profile API sharepoint](https://dvagov.sharepoint.com/sites/OITEPMOVAPROPUB/SitePages/Getting-Started.aspx)
- [Health Benefits Swagger Doc](https://qa.vaprofile.va.gov:7005/health-benefit/swagger-ui/index.html?configUrl=/health-benefit/v3/api-docs/swagger-config)
- [Personal Information](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/personal_information/backend_documentation.md)
- [Contact Information](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/contact_information/backend_documentation.md)
- [Military Information](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/military_info/backend_documentation.md)
- [Direct Deposit Information](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/direct_deposit/backend_documentation.md)
- [Notification Settings](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/notification_settings/backend_documentation.md)
- [Account Security](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/products/identity-personalization/profile/account_security/backend_documentation.md)

**Comment by Agile6MSkinner** [comment]
This functionality was built but the ticket was never closed. Likely it was worked in a dupe.
